src/modules/attributes.ts

```diff
@@ -28,18 +32,18 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
         elm.setAttribute(key, "");
       } else if (cur === false) {
         elm.removeAttribute(key);
+      } else if (key.charCodeAt(0) !== xChar) {
+        elm.setAttribute(key, cur as any);
+      } else if (key.charCodeAt(3) === colonChar) {
+        key.charCodeAt(0) === sChar
```

Comment:
This check will _always_ be `false`. If the first character was "s" then `key.charCodeAt(0) !== xChar` check above would've succeeded and we would not be in this branch. That check basically assumes that all special attributes begin with an "x".

Reply:
how embarrassing, you are right :)

Reply:
the commit was amended so that the first condition was removed and the comments are restored with mention of the extra namespaces. Detection of the 's' char was removed and both conditions now detect the xml 'm' char
```javascript
      } else if (key.charCodeAt(3) === colonChar) {
        // Assume 'xml' or 'svg' namespace
        key.charCodeAt(1) === mChar
          ? elm.setAttributeNS(xmlNS, key, cur as any)
          : elm.setAttributeNS(svgNS, key, cur as any);
      } else if (key.charCodeAt(5) === colonChar) {
        // Assume 'xmlns' or 'xlink' namespace
        key.charCodeAt(1) === mChar
          ? elm.setAttributeNS(xmlnsNS, key, cur as any)
          : elm.setAttributeNS(xlinkNS, key, cur as any);
```